qiskit/algorithms/optimizers/optimizer.py

```diff
@@ -212,6 +240,11 @@ def optimize(
             )
         pass
 
+    @property
+    def settings(self) -> Dict[str, Any]:
+        """The optimizer settings in a dictionary format."""
+        raise NotImplementedError("Dictionary conversion not implemented for this optimizer.")
```

Comment:
I think this needs to be removed since there's another `settings` at line 162

Reply:
yep, I missed that one, it should already be updated 🙂